src/nodes.coffee

```diff
@@ -3026,10 +3026,20 @@ exports.For = class For extends While
       begin = new Value new Literal "_begin"
       end = new Value new Literal "_end"
       positive = new Value new Literal "_positive"
+      stepVal = new Value new Literal "_step"
```

Comment:
With using `freeVariable` and `icedlocal` below, it's a little bit of belt and suspenders.